aiohttp/web_response.py

```diff
@@ -259,9 +259,17 @@ def last_modified(
             self._headers.pop(hdrs.LAST_MODIFIED, None)
         elif isinstance(value, (int, float)):
             self._headers[hdrs.LAST_MODIFIED] = time.strftime(
-                "%a, %d %b %Y %H:%M:%S GMT", time.gmtime(math.ceil(value))
+                "%a, %d %b %Y %H:%M:%S GMT", time.gmtime(math.floor(value))
             )
         elif isinstance(value, datetime.datetime):
+            if not value.tzinfo:
```

Comment:
There is no easy way to create tz-aware local current time.
In turn, `datetime.now()` is widely used as *local naive time*. 
`datetime.utcnow()` usage is discouraged for the sake of `datetime.now(tz=timezone.utc)`.

So, I suggest dropping this warning but replacing `value.utctimetuple()` with `value.timetuple()` a few lines below.

https://docs.aiohttp.org/en/stable/web_reference.html?highlight=last_modified#aiohttp.web.StreamResponse.last_modified should describe how the library works with naive time objects.

Reply:
So, I guess there are 3 cases in new behaviour:

- Naive `datetime` is used.
Must be considered as UTC. Documentation clearly describes this case.

- tz-aware `datetime` is used and timezone is `utc`.
All is ok, user gave us correct object.

- tz-aware `datetime` is used and timezone **is not** `utc`.
That's definitely wrong input to use with `value.timetuple()`. `ValueException` should be raised.

Is that correct?